modules/tile-converter/src/i3s-converter/helpers/geometry-converter.js

```diff
@@ -21,7 +21,7 @@ const OBJECT_ID_TYPE = 'Oid32';
 * 'CUSTOM_ATTRIBUTE_2' - Attribute name which includes batch info and used by New York map.
 * _BATCHID - default attribute name which includes batch info.
 */
-const BATCHED_ID_POSSIBLE_ATTRIBUTE_NAMES = ['CUSTOM_ATTRIBUTE_2', '_BATCHID'];
+const BATCHED_ID_POSSIBLE_ATTRIBUTE_NAMES = ['CUSTOM_ATTRIBUTE_2', '_BATCHID', 'BATCHID'];
```

Comment:
Maybe add comment for each value, stating that this value is for older version.

Reply:
Should this not go into the 3D Tile loader rather than the tile-converter? Can we normalize this in the loader so we don't need to deal with it here?